src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -195,12 +212,13 @@ NavHeader.propTypes = {
   hideSeeMember: PropTypes.bool,
   animatedHeight: PropTypes.number,
   opacityHeaderAnimation: PropTypes.number,
+  opacityImage: PropTypes.number,
   getTopicDetail: PropTypes.func,
   setMemberCount: PropTypes.func,
   memberCount: PropTypes.number,
   topicDetail: PropTypes.object,
   setIsFollow: PropTypes.func,
-
+  isLoading: PropTypes.bool,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes is a good practice for type checking in React. This helps catch bugs by validating the types of props being passed to components.



<!-- This is an auto-generated comment by OSS CodeRabbit -->